appveyor.yml

```diff
@@ -0,0 +1,25 @@
+pull_requests:
+  do_not_increment_build_number: true
```

Comment:
@fir3pho3nixx we should set this in Castle Core too, pull request build numbers are throw away so might as well get a random string appended to the end of the version.